app/presenters/pro_resource_presenter.rb

```diff
@@ -37,9 +37,15 @@ def has_displayable_tags?
     displayable_tags.present?
   end
 
-  # We omit from view the "culturelover" internal-use tag
+  # We omit from view internal-use tags starting "culturelover"
   def displayable_tags
-    resource.taxonomy[:tags].reject { |_pro_path, tag| tag == 'culturelover' }
+    return nil unless resource.has_taxonomy?(:tags)
+    @displayable_tags ||= resource.taxonomy[:tags].reject { |_pro_path, tag| tag.start_with?('culturelover') }
```

Comment:
might make sense for this to make use of pro_json_api_filters[:tags] to allow more flexibility 
not sure if that's always guaranteed accessible at the moment since that's defined in the pro_json_api_consumer concern